index.html

```diff
@@ -1734,39 +1812,113 @@ <h3>Document Encoding</h3>
             for processing, in which case the serialisation requirements do not apply.</p>
       </section>
       <section>
-        <h3>Foreign Elements and Attributes</h3>
-        <p>
-          A <a>DAPT Document</a> MAY contain elements and attributes that are neither specifically permitted nor forbidden by a profile.
-        </p>
-        <p class ="note">
-          <a>DAPT Documents</a> remain subject to the content conformance requirements specified at Section 3.1 of [[TTML2]].
-          In particular, a <a>DAPT Document</a> can contain elements and attributes not in any TT namespace, i.e. in foreign namespaces, since such elements and attributes are pruned by the algorithm at Section 4 of [[TTML2]] prior to evaluating content conformance.
+        <h3>Processing of unrecognised or foreign elements and attributes</h3>
+        <p>The requirements in this section are intended to facilitate forwards and backwards compatibility,
+          specifically to permit:
         </p>
-        <p class ="note">
-          For validation purposes it is good practice to define and use a content specification for all foreign namespace elements and attributes used within a <a>DAPT Document</a>.
-        </p>
-        <p>
-          A <a>transformation processor</a> SHOULD preserve such elements or attributes whenever possible.
-        </p>
-        <p class="ednote">Do we need to say that a <a>presentation processor</a> may ignore foreign vocab?</p>
+        <ul>
+          <li>DAPT processors targeted at one version of the specification to process <a>DAPT documents</a> that
+            include vocabulary or semantics defined in future versions,
+            albeit without supporting the later features;</li>
+          <li>DAPT processors targeted at one version of the specification to process <a>DAPT documents</a>
+            authored for an earlier version with similar or identical behaviour to a DAPT processor
+            targeted at that earlier version.</li>
+        </ul>
+        <p>A <a>DAPT document</a> that conforms to a future version of the specification could specify
+          conformance to multiple DAPT <a>content profiles</a>.</p>
+        <section id="unrecognised-elements-and-attributes">
+          <h4>Unrecognised vocabulary</h4>
+          <p><dfn>Unrecognised vocabulary</dfn> is the set of elements and attributes that are
+            not associated with <a>features</a> that the processor supports.</p>
```

Comment:
Instead of linking "features" to its definition in TTML2, could we link to section E.1 which actually lists the features (and points to TTML2)?

Reply:
Done in 1e6bd9b.